### PR TITLE
bump to v5.1.1 fix SmartRate response JSON format

### PR DESCRIPTION
**Pull request by jontsai**

### CODE REVIEW

**Review by Justintime50 — APPROVED** [review]

**Review by Roguelazer — CHANGES_REQUESTED** [review]
this is missing `version.py`.

**Review by Justintime50 — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -23,7 +24,7 @@
 
 setup(
     name='easypost',
-    version='5.1.0',
+    version='5.1.1',
```
Comment (pseudoben) [inline setup.py:27]: It seems like it's too easy for this to get out of sync with version.py. Maybe this should be automatically populated?

**Review by pseudoben — APPROVED** [review]
Suggestion of auto-populating version from version.py is non-blocking.

File: setup.py (hunk 1 above)
Comment (jontsai) [reply setup.py:27]: Good suggestions. Perhaps a unit test or something.

An earlier version was trying to to some introspection, but it messed with packaging/setuptools.

**Review by Roguelazer — APPROVED** [review]

**Pull request merged**